### DISCUSSION TITLE
GitHub Mobile shows old code

### DISCUSSION THREAD

**Discussion by ghost** [community/community#147458] [Mobile]
Just realized, that after editing via GitHub WebUI, the code of https://github.com/sajde/workflows didn't update within GitHub Mobile. I already tried logging out and in again and deleted the cache. Didn't work out.

I just tried adding a new file via GitHub Mobile (newfile.txt) and after committing, I can see it via WebUI, but NOT via GitHub Mobile...

<details>
<summary>ℹ️ GitHub Mobile doesn't update.</summary>
GitHub for iOS, Version 1.190.0 (149394517)
Device: iPhone 15 Pro (iOS 17.6.1)
</details>

**Comment by eliperkins** [comment]
Hey @sajde, thanks for this feedback! It definitely sounds like a frustrating bug to not have GitHub Mobile reflect the source of truth for this repo.

To help me debug further, can you record a video showing the steps to reproduce this? That will help me and the engineering team here get this bug ironed out.

Thanks for helping improve GitHub Mobile! 💪

**Reply by ghost** [reply]
Hi @eliperkins , thanks for your reply! I just uploaded a short video: https://github.com/sajde/workflows/blob/441d722da9cd342710d4091f2efb0573f8d54071/github-issue.mp4
